From c6f568ad1585bce2526f1c7275effe6a28673b2c Mon Sep 17 00:00:00 2001
From: Julien Thomas <jthomas@zenetys.com>
Date: Thu, 18 Apr 2024 12:15:01 +0200
Subject: [PATCH] Revert requirement on go 1.21

Change was introduced in commit bbe196b "feat: upgrade
go.mod to 1.21 (#1830)".
---
 go.mod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index 78d7a21..1dd5c2c 100644
--- a/go.mod
+++ b/go.mod
@@ -30,4 +30,4 @@ require (
 	golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2 // indirect
 )
 
-go 1.21
+go 1.20
-- 
2.21.1

